src/styles/Theme.tsx

```diff
@@ -16,58 +16,58 @@ const theme: DefaultTheme = {
     main: 'Inter',
     fontSize: {
       body: {
-        min: '20px',
-        size: '24px',
-        max: '2rem',
+        min: '12px',
+        size: '1.25vw',
+        max: '18px',
         weight: '300',
         height: '',
       },
       input: {
-        min: '24px',
-        size: '36px',
-        max: '2.4rem',
+        min: '16px',
+        size: '1.25vw',
```

Comment:
values here should be scale according to the range; ie if 1.25vw for 12px then for 16px it should be something higher than 1.25, can be like 1.27 etc; should scale too

Reply:
6f83ddcef630951d836d70309f969b0953101e6e